apricot/local/serviceutil.go

```diff
@@ -84,7 +87,19 @@ func (s *Service) getStringMap(path string) map[string]string {
 			if v.Type() != cfgbackend.IT_Value {
 				continue
 			}
-			theMap[k] = v.Value()
+			if v.Value() != strings.TrimSpace(v.Value()) {
+				valueToStore := v.Value()
+				log.WithError(errors.New("leading or trailing space in value")).
+					WithField("path", path).
+					WithField("key", k).
+					Warning("Consul var has leading/trailing space in its value")
```

Comment:
This part (the IL message) depends on the viper setting.
If false: `Warning("found unsanitized string value in configuration backend, returning anyway")`
If true: `Warning("found unsanitized string value in configuration backend, returning whitespace-trimmed string")`

InfoLogger will then append the `error` message as well.